### PR TITLE
Drop auth in node/rhvh kickstart

### PR DESCRIPTION
**Pull request by michalskrivanek**
legacy "auth" is deprecated

### CODE REVIEW

File: node.ks.in [hunk 1]
```diff
@@ -12,7 +12,7 @@ fi
 timezone --utc UTC
 lang en_US.UTF-8
 keyboard us
-auth --enableshadow --passalgo=sha512
+authselect minimal
```
Comment (almusil) [inline node.ks.in:15]: `authselect select minimal`

Comment (michalskrivanek) [reply node.ks.in:15]: or drop the whole thing? it apparently worked like this too...:)

Comment (almusil) [reply node.ks.in:15]: I don't think it worked, or at least there was completely different profile and only on RHVH

Comment (michalskrivanek) [reply node.ks.in:15]: can't fins ks logs, but i booted it and the profile is set correctly to minimal anyway, likely from the rhvh's own ks. So i'll drop the whole thing here then...

Comment (michalskrivanek) [reply node.ks.in]: works for rhvh, for node it likely needs https://github.com/oVirt/ovirt-node-ng-image/pull/10

**Comment by michalskrivanek** [comment]
with https://github.com/oVirt/ovirt-node-ng-image/pull/10:
[root@ost-basic-suite-master-host-1 ~]# authselect current
Profile ID: minimal
Enabled features: None

**Pull request merged**